src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -245,21 +246,23 @@ class CrossDeviceLinkUI extends Component {
             { invalidNumber && <div className={style.numberError}>{translate('errors.invalid_number.message')}</div> }
           </div>
           <div className={style.copyLinkSection}>
-            <div className={`${style.label}`}>{translate('cross_device.link.copy_link_label')}</div>
-              <div className={classNames(style.linkContainer, {[style.copySuccess]: this.state.copySuccess})}>
-                <textarea className={style.linkText} value={mobileUrl} ref={(element) => this.linkText = element}/>
-                { document.queryCommandSupported('copy') &&
-                  <div className={style.actionContainer}>
-                    <button
-                      type="button"
-                      onClick={this.copyToClipboard}
-                      className={style.copyToClipboard}
-                    >
-                      {linkCopy}
-                    </button>
-                  </div>
-                }
-              </div>
+            <div tabIndex="0" className={style.label}>{translate('cross_device.link.copy_link_label')}</div>
+            <div className={classNames(style.linkContainer, this.state.copySuccess && style.copySuccess)}>
+              <textarea readonly tabIndex="0" className={style.linkText} ref={(element) => this.linkText = element}>
```

Comment:
is it `tabIndex` or `tabindex`?
Also, if `tabIndex="0"` is the default navigation, is it really necessary?

Reply:
We seem to be a bit inconsistent with how it's written in the repo, I've found both forms used in different components. But React documentation says "the HTML attribute `tabindex` corresponds to the attribute `tabIndex` in React."
But in terms of necessity, the label needs it, but not the textarea. So have removed that one.